public/index.html

```diff
@@ -119,7 +119,7 @@
         <div id="frameMain">
             <div id="frameContentWin">
                 <div id="frameTitleDock">
-                    <span id="frameTitleText">Elundus Core - V0.0.0</span>
+                    <span id="frameTitleText">Elundus Core - V0.2.0</span>
```

Comment:
The version is something I'll someday probably forget updating when making a new release, but will do for now.